draft-rescorla-tls-esni.md

```diff
@@ -603,6 +622,15 @@ verifying the server's identity in its certificate.
 odd, and probably some precise rules about handling ESNI and no
 SNI uniformly?]]
 
+## Size of Anonymity Sets
+
+The domains, destinations and ESNI key of an anonymity set are publicly known.
+The strength of the anonymity provided by ESNI is directly related to the number of
+domains sharing both an ESNI key and destination A (or AAAA) records.
+
+Providers SHOULD minimize the number of anonymity sets they support. A single large set is
+better than many small sets.
+A single ESNI-protected domain served by a separate provider server does not provide anonymity.
```

Comment:
I think you need to soften this point. It's true in principle, but plenty of monitoring entities will not have access to this information.

Reply:
softened.